src/main/java/org/openstreetmap/atlas/checks/commands/FlagStatisticsSubCommand.java

```diff
@@ -111,32 +116,30 @@ public int execute()
                 final Map<String, Map<String, Counter>> differenceCounts = this
                         .getDifference(referenceCounts, inputCounts);
 
-                // Write outputs for the reference and difference if requested
-                if (outputTypes.contains("full"))
+                // Write outputs for the difference if requested
+                if (outputTypes.contains(FULL))
                 {
-                    this.writeCSV(outputFolder + "/reference.csv",
-                            generateFullOutput(referenceCounts));
-                    this.writeCSV(outputFolder + "/difference.csv",
+                    this.writeCSV(outputFolder + "/runSummaryDifference.csv",
                             generateFullOutput(differenceCounts));
                 }
 
                 // Add the reference and difference metrics to the totals output
-                totalsOutput = generateTotalsOutputStage2(totalsOutput, referenceCounts,
-                        differenceCounts);
+                totalsOutput = addReferenceAndDifferenceToTotalsOutput(totalsOutput,
+                        referenceCounts, differenceCounts);
                 // Add the reference and difference metrics to the counts output
-                countsOutput = generateCountsOutputStage2(countsOutput, referenceCounts,
-                        differenceCounts);
+                countsOutput = addReferenceAndDifferenceToCountsOutput(countsOutput,
+                        referenceCounts, differenceCounts);
             }
 
             // Write the totals output if requested
             if (outputTypes.contains("totals"))
             {
-                this.writeCSV(outputFolder + "/totals.csv", totalsOutput);
+                this.writeCSV(outputFolder + "/checkSummary.csv", totalsOutput);
```

Comment:
Should we change the output type name as well? Maybe we can put all of the supported `output-types` in a enum set

Reply:
Done